app/config/AppConfig.scala

```diff
@@ -48,4 +50,6 @@ class AppConfig @Inject()(
 
   lazy val customsDeclarationsGoodsTakenOutOfEuUrl: String = loadConfig("urls.customsDeclarationsGoodsTakenOutOfEu")
   lazy val serviceAvailabilityUrl: String = loadConfig("urls.serviceAvailability")
+
+  lazy val cacheTimeToLive: FiniteDuration = servicesConfig.getDuration("mongodb.timeToLive").asInstanceOf[FiniteDuration]
```

Comment:
it is more ile-query related option rather than mongodb

Reply:
Agree with the above

Reply:
Changed